README.md

```diff
@@ -1,14 +1,28 @@
 # Frisky: A general purpose Slack bot written in Python
 
-## Running Frisky locally
-Frisky uses `pipenv` to manage its Python environment. From the project directory, you can run `pipenv install` to get your environment set up, and then `pipenv run python manage.py runserver`
+1. [Running Frisky Locally](#running-frisky-locally)
+2. [Writing a Plugin](#writing-a-plugin)
+   1. [An example plugin](#an-example-plugin)
+3. [Troubleshooting](#troubleshooting)
 
-Because Frisky is event driven, you will need to emulate events coming from Slack in order to test your featured end-to-end. However, a test case class is provided to more easily test simple plugins.
+## Running Frisky Locally
+Before you get started, make sure you have python 3.8.1 installed.  Frisky is a modern ~~cat~~ bot with modern needs!
+
+Frisky uses `pipenv` to manage its Python environment. This is just a standard pip package that can be installed with
+`pip install pipenv`, the python runtime you install it to does not matter.  From the project directory, you can run
+`pipenv install` to get your environment set up, and then `pipenv run python manage.py runserver`
+
+Because Frisky is event driven, you will need to emulate events coming from Slack in order to test your featured
+end-to-end. However, a test case class is provided to more easily test simple plugins.
 
 ## Writing a plugin
-To create a plugin, add a python file underneath the `plugins/` directory. Inside this file, add a function called `handle_message` with the signature: `handle_message(*args, **kwargs) -> str:`. The name of your file determines the command name that will invoke it via Slack, so sending a `?ping` command in Slack would be handled by `ping.py`.
+To create a plugin, add a python file underneath the `plugins/` directory. Inside this file, add a function called
+`handle_message` with the signature: `handle_message(*args, **kwargs) -> str:`. The name of your file determines the
+command name that will invoke it via Slack, so sending a `?ping` command in Slack would be handled by `ping.py`.
 
-The Slack event handler will parse incoming messages in the form `?command foo bar xyzzy` and pass everyhing after the command name to the plugin as a list to the `*args` parameter. The `**kwargs` parameter will be used for any extra context to pass along to the bot, currently this is just the channel name as `channel`.
+The Slack event handler will parse incoming messages in the form `?command foo bar xyzzy` and pass everything after the 
```

Comment:
Likewise, we have concrete Event classes now, and we should probably mention this